scripts/4-generateIndexFiles.ts

```diff
@@ -75,31 +76,33 @@ async function generateIconIndex(iconNames: string[], type: 'print' | 'pop' | 'p
     (name) => `export { default as ${camelCase(name)} } from './${name}'`,
   )
   const content = `${iconExports.join('\n')}`
-  const path = PATH_PEPICONS + `/src/icons/${type}/index.ts`
-  writeFile(path, content)
+  const filePath = path.resolve(PATH_PEPICONS, `./src/icons/${type}/index.ts`)
+  writeFile(filePath, content)
 }
 
 export async function generateIndexFiles() {
-  const popFilePaths = await listFiles(PATH_ROOT + `/scripts/export/`, /.+pop\/.+\.svg/gi)
-  const popFilenames = popFilePaths.map<string>((folder) => folder.split('/').slice(-2).join('/'))
+  const popFilePaths = await listFiles(
+    path.resolve(PATH_ROOT, `./scripts/export/`),
+    /.+pop[/\\].+\.svg/gi,
+  )
+  const popFilenames = popFilePaths.map<string>((folder) =>
+    folder.split(path.sep).slice(-2).join(path.sep),
+  )
   const iconNames = popFilenames
     .map(filenameToIconName)
-    .map((name) => [
+    .flatMap((name) => [
       name,
       `${name}-circle-off`,
       `${name}-circle`,
       `${name}-circle-filled`,
       `${name}-off`,
     ])
-    .flat()
   const iconSynonyms = popFilenames
     .map(filenameToIconSynonyms)
-    .map((synonyms) => [synonyms, [], [], [], []])
-    .flat()
+    .flatMap((synonyms) => [synonyms, [], [], [], []])
```

Comment:
```suggestion
    .flatMap((synonyms) => synonyms)
```
Not sure why I was doing all of this before and then still executing `.flat()` at the end 😂  But I think this should basically be the same since we flatten it?

Reply:
`.flat` is `.flat(1)` so if you had 
`["foo", "bar"]` your result would be `['foo', [], [], [], [], 'bar', [], [], [], []]` in both the original and my change. With your change you'd only have `['foo', 'bar']`